mirgecom/steppers.py

```diff
@@ -27,9 +27,47 @@
 THE SOFTWARE.
 """
 
+from logpyle import (LogManager, add_general_quantities,
+        add_simulation_quantities, add_run_info, IntervalTimer,
+        set_dt, LogQuantity)
+
+class Pressure(LogQuantity):
+  def __init__(self, discr, current_state, eos):
+        LogQuantity.__init__(self, "pressure", "p")
+
+        from mirgecom.euler import split_conserved
+        cv = split_conserved(discr.dim, current_state)
+        self.dv = eos.dependent_vars(cv)
+
+        self.discr = discr
+
+  def __call__(self):
+    from functools import partial
+    _min = partial(self.discr.nodal_min, "vol")
+    _max = partial(self.discr.nodal_max, "vol")
+
+    return _min(self.dv.pressure)
+
+class Temperature(LogQuantity):
+  def __init__(self, discr, current_state, eos):
+        LogQuantity.__init__(self, "temperature", "K")
+
+        from mirgecom.euler import split_conserved
+        cv = split_conserved(discr.dim, current_state)
+        self.dv = eos.dependent_vars(cv)
+
+        self.discr = discr
+
+  def __call__(self):
+    from functools import partial
+    _min = partial(self.discr.nodal_min, "vol")
```

Comment:
Make sure that there's no MPI reduction in this.

Reply:
As far as I can see, there is none.